app/src/pages/trace/SpanDetails.tsx

```diff
@@ -1331,11 +1331,13 @@ function LLMMessage({ message }: { message: AttributeMessage }) {
                     `}
                   >
                     {toolCall?.function?.name as string}(
-                    {JSON.stringify(
-                      JSON.parse(toolCall?.function?.arguments as string),
-                      null,
-                      2
-                    )}
+                    {toolCall?.function?.arguments
```

Comment:
You could go one step farther and ensure it is a string if you want

```suggestion
                    {typeof toolCall?.function?.arguments === "string"
```